src/styles/block/calendar.scss

```
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  gap: $gap;
+  width: $width;
+  flex-wrap: wrap;
+
+  padding: 50px $calendar-padding 0;
+
+  @each $day, $number in $days-of-week {
+    &--start-day-#{$day} {
+      & :first-child {
+        margin-left: (#{($number) * ($calendar-day-width + 1px)});
+      }
+    }
+  }
+
+  @for $index from 28 through 31 {
+    &--month-length-#{$index} {
+      & :nth-child(1n + #{$index + 1}) {
+        display: contents;
```

you should not display it at all:
```suggestion
        display: none;
```